cs251tk/student/markdownify/markdownify.py

```diff
@@ -31,15 +34,20 @@ def markdownify(spec_id, username, spec, basedir, debug):
             result = process_file(filename, steps, options, spec, cwd, supporting)
             results['files'][filename] = result
 
+        # now we remove any compiled binaries
         try:
             for file in spec['files']:
                 os.remove('{}.exec'.format(file['filename']))
+        except FileNotFoundError:
+            pass
+
+        # and we remove any supporting files
+        try:
```

Comment:
Sounds right, I believe it is best to extract the main places where errors will occur out into as much error handling as possible.  In this case, two different sets of possibly-breaking code are separated.

However, since the two are